static/app/views/settings/projectAlerts/issueRuleEditor/ruleNode.tsx

```diff
@@ -415,3 +419,7 @@ const DeleteButton = styled(Button)`
 const MarginlessAlert = styled(Alert)`
   margin: 0;
 `;
+
+const StyledFeatureBadge = styled(FeatureBadge)`
+  margin: 0 ${space(1)} 0 0;
```

Comment:
Can you just use `margin-right`?

Reply:
in this case i also wanted to override the margin as well

Reply:
Ah it has a default? ew

Reply:
very ew, most of our badges dont